third_party/move/move-vm/runtime/src/move_vm.rs

```diff
@@ -65,6 +70,22 @@ impl MoveVM {
         Session {
             move_vm: self,
             data_cache: TransactionDataCache::new(remote),
+            module_store: ModuleStorageAdapter::new(self.runtime.module_storage()),
```

Comment:
I see that you are trying to not break the old APIs. Just wondering what will actually take us to get rid of this? I assume it'll be a rather big refactoring to move the module storage into the aptos vm?

Reply:
Indeed. The plan is we would use a wrapper MVHashMap to implement this module storage inside AptosVM. I would doubt if we can simply remove such API tho if others would like to use this repo.